docs/source/user_manual/chaco_tutorial.rst

```diff
@@ -100,57 +100,15 @@ drop-down menus (not show here), the user can change plot attributes like the
 colormap and the number of contour levels used in the center plot, as well as
 the actual function being plotted.
 
-
-Script-oriented plotting
-========================
```

Comment:
I think it is stil worth comparing/contrasting these two styles - script-oriented vs application-oriented - so we would be using matplotlib instead of chaco.shell to demonstrate the script-oriented style of plotting.